examples/plotting/magnetogram_active_regions.py

```diff
@@ -14,27 +14,28 @@
 
 import sunpy.coordinates
 import sunpy.map
+import sunpy.data.sample
 from sunpy.io.special import srs
 from sunpy.net import Fido
 from sunpy.net import attrs as a
 from sunpy.time import parse_time
 
 ##############################################################################
-# For this example, we will search for and download a single HMI using Fido.
-start_time = parse_time("2017-01-25")
-end_time = start_time + TimeDelta(23*u.hour + 59*u.minute + 59*u.second)
-results = Fido.search(a.Time(start_time, end_time),
-                      a.Instrument.hmi & a.Physobs.los_magnetic_field,
-                      a.Sample(60 * u.second))
+# For this example, we will start with the sample data. We need an HMI
+# file and use it to create a map.
+
+smap = sunpy.map.Map(sunpy.data.sample.HMI_LOS_IMAGE)
 
 ##############################################################################
-# Let's select only the first file, download it and create a map.
-result = results[0, 0]
-file_name = Fido.fetch(result)
-smap = sunpy.map.Map(file_name)
+# Then extract the observation time of the sample image. We use it to
+# calculate the time interval for the SRS file search.
+
+start_time = parse_time(smap.date)
+end_time = start_time + TimeDelta(23*u.hour + 59*u.minute + 59*u.second)
```

Comment:
```suggestion
end_time = start_time + TimeDelta(23*u.day)
```
Can we not do this instead?

Reply:
No, SRS files are published at the same time each day. I checked this, TimeDelta(1*u.day) gives two files

Reply:
So it does work we just need to index out the first file? 

Can you add a comment as to why we had to be very specific with this timedelta?

Reply:
> No, SRS files are published at the same time each day. I checked this, TimeDelta(1*u.day) gives two files

Good catch!

Reply:
> So it does work we just need to index out the first file?
> 
Yes, it works. We chose the first file later:
![image](https://user-images.githubusercontent.com/17323271/98945878-cbd42000-24fb-11eb-9ed7-e548f018cacf.png)

> Can you add a comment as to why we had to be very specific with this timedelta?
Perhaps then it is appropriate to replace the `srs_downloaded_files` list with `srs_downloaded_file` variable and then add such a comment about this timedelta?

Reply:
Sounds good to me.

Reply:
No, `Fido.fetch()` returns a list object, so I leave it with index access. Only adding a comment